### PR TITLE
Convert stub docs pages to MyST Markdown

### LINKED ISSUE #182452: [Docathon] Convert 4 stub/redirect rST pages to MyST Markdown
## Description

These four `.rst` files are very small — redirects, deprecation notices, or external pointers — and can all be handled in a single PR.

| File | What it is |
|------|-----------|
| [`docs/source/notes/custom_operators.rst`](https://github.com/pytorch/pytorch/blob/main/docs/source/notes/custom_operators.rst) | Redirect (HTML meta-refresh to tutorials site) |
| [`docs/source/notes/cpu_threading_torchscript_inference.rst`](https://github.com/pytorch/pytorch/blob/main/docs/source/notes/cpu_threading_torchscript_inference.rst) | Deprecation stub (TorchScript → torch.export) |
| [`docs/source/elastic/examples.rst`](https://github.com/pytorch/pytorch/blob/main/docs/source/elastic/examples.rst) | External pointer (3 lines linking to GitHub README) |
| [`docs/source/elastic/kubernetes.rst`](https://github.com/pytorch/pytorch/blob/main/docs/source/elastic/kubernetes.rst) | External pointer (4 lines linking to GitHub README) |

## Steps

1. **Assign the issue** to yourself using the `/assigntome` comment.

2. **Rename each file using `git mv`:**
   ```bash
   git mv docs/source/notes/custom_operators.rst docs/source/notes/custom_operators.md
   git mv docs/source/notes/cpu_threading_torchscript_inference.rst docs/source/notes/cpu_threading_torchscript_inference.md
   git mv docs/source/elastic/examples.rst docs/source/elastic/examples.md
   git mv docs/source/elastic/kubernetes.rst docs/source/elastic/kubernetes.md
   ```
   **Important!** Use `git mv` to convert the files — DO NOT delete the `.rst` and create new `.md` files.
   We won't accept PRs with deleted `.rst` files. We want to preserve the git log.

3. **Convert the content** from rST to MyST Markdown syntax. These files are very small so manual conversion is straightforward.

4. **Build the docs locally** as described in the [README](https://github.com/pytorch/pytorch/tree/main/docs).
   Preview locally with `make serve` to ensure the HTML renders correctly.

5. **Run lintrunner** from the root of the repository:
   ```bash
   pip install lintrunner
   lintrunner init
   lintrunner -m main
   ```

6. **Submit a PR** with your changes and ensure the PR passes the doc CI tests. Specifically it needs to pass:
   - `pull / linux-docs / build-docs-python-false (pull_request)`
   - `linux-jammy-py3.9-gcc11 / test (docs_test, 1, 1, lf.linux.2xlarge)`

   Doc tests start after the Docker build, which might take about 30 minutes.
   The doc build itself should take about 25 minutes. When the build is done,
   you can see the preview of your build by clicking
   **📄 Preview Python docs built from this PR** in the pytorch-bot comment.

7. **Label your PR** with: `docathon-2026`, `module: docs`.

8. **Compare the preview** of the pages you converted with the same pages on the main branch.
   The converted pages in preview must look the same as in the main branch.

9. Once the PR is accepted, one of the maintainers will merge it.

## References

- [Quick MyST Conversion Reference](https://github.com/pytorch/pytorch/wiki/Docstring-Guidelines#restructuredtext-vs-markdown) (wiki)
- [MyST Parser documentation](https://myst-parser.readthedocs.io/en/v0.15.1/sphinx/use.html)
- [Example PR](https://github.com/pytorch/pytorch/pull/154438)

cc @sekyondaMeta @AlannaBurke

### PR DESCRIPTION
**Pull request by Vasanthadithya-mundrathi**
Fixes #182452

## Summary
- Rename four small stub/redirect docs pages from `.rst` to `.md`.
- Convert headings, links, the TorchScript warning, and the raw HTML redirect to MyST Markdown syntax.
- Keep the rename-only commit separate from the syntax conversion to preserve file history.

## Test plan
- `git diff --check`
- `PATH=/tmp/docathon-torch-venv/bin:$PATH lintrunner -m upstream/main`
- Minimal Sphinx/MyST build of the four converted pages with `-W --keep-going`

cc @svekars @sekyondaMeta @AlannaBurke

### CODE REVIEW

**Comment by Vasanthadithya-mundrathi** [comment]
Could a maintainer add the Docathon labels (`docathon-2026`, `module: docs`, `easy`, `rst-to-md`) and `topic: not user facing` when convenient? I do not have permission to apply repository labels.

**Comment by svekars** [comment]
@claude

**Review by svekars — APPROVED** [review]
